pkg/syfs/syfs.go

```diff
@@ -0,0 +1,64 @@
+// Pakcage syfs provides functions to access singularity's file system
+// layout
```

Comment:
Is there that copyright thing thats supposed to go here?

Reply:
Right! Fixing.